File: solutions/beecrowd/1009/1009.clj
(ns main)

(defn main []
  (let [_ (read-line)
        a (Double/parseDouble (read-line))
        b (Double/parseDouble (read-line))]
    (format "%.2f" (+ a (* b 0.15)))))

(printf "TOTAL = R$ %s%n" (main))

File: solutions/beecrowd/1009/1009.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`);

    sc.next();
    val b: Double = sc.next().toDouble();
    val c: Double = sc.next().toDouble();

    println("TOTAL = R$ %.2f".format(b + (c * 0.15)));
}
